components/applications-service/pkg/server/server.go

```diff
@@ -260,6 +260,26 @@ func (app *ApplicationsServer) GetServicesStats(c context.Context,
 	}, nil
 }
 
+// GetDisconnectedServices returns a list of disconnected services
+func (app *ApplicationsServer) GetDisconnectedServices(c context.Context,
+	request *applications.DisconnectedServicesReq) (*applications.ServicesRes, error) {
+
+	thresholdMinutes := request.GetThresholdMinutes()
+	if thresholdMinutes <= 0 {
+		return new(applications.ServicesRes),
```

Comment:
Any reason to prefer this to `nil` like we return in the error case below?

Reply:
No preference, just a standard I have followed, I'm willing to change if there is any benefit 🤔

Reply:
No big benefit as far as I know (I suppose we save an allocation, but I'm skeptical it would be noticable), just noticed it because it is different than the error case a few lines below.

Reply:
You're right, thanks for pointing out the mismatch, I will fix it!